services/gundeck/src/Gundeck/Push/Native/Serialise.hs

```diff
@@ -38,24 +37,24 @@ serialise m a = do
   rs <- prepare m a
   case rs of
     Left failure -> return $! Left $! failure
-    Right (v, prio, aps) -> case renderText (a ^. addrTransport) aps prio v of
+    Right (v, prio) -> case renderText (a ^. addrTransport) prio v of
       Nothing -> return $ Left PayloadTooLarge
       Just txt -> return $ Right txt
 
-prepare :: NativePush -> Address -> IO (Either Failure (Value, Priority, Maybe ApsData))
+prepare :: NativePush -> Address -> IO (Either Failure (Value, Priority))
 prepare m a = case m of
-  NativePush nid prio aps ->
+  NativePush nid prio _aps ->
```

Comment:
I'm tempted to remove this, but let's keep this PR small, there is also https://github.com/wireapp/wire-server/pull/1209.